scripts/validate.py

```diff
@@ -16,6 +19,62 @@
         validate.py ./cfr.json ./schema/cfr.schema.json
 """
 
+def intersect_op(listA, listB):
+    set(listA).intersection(listB)
+
+def validate_targeting(targeting):
+    jexl = JEXL()
```

Comment:
Looks like we just need one JEXL instance here, let's move `jexl` and `context` out of this function to the module level.

Reply:
Just notice if we move all the JEXL creation bit out, this function only has the last line left. Looks like we can just refactor it out by directly calling `jexl.evaluate(targeting)` in `validate_item_targeting`.